Imports/ArcGIS/Runtime/Toolkit/Dialogs.2.0/AuthenticationView.qml

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright 2012-2016 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ ******************************************************************************/
+
+import QtQuick 2.5
+
+Item {
+    id: authView
+    anchors.fill: parent
+    property var authenticationManager
+    property var authChallenge
+
+    // Connect to the target (AuthenticationManager)
+    // authenticationChallenge signal
+    Connections {
+        target: authenticationManager
+
+        onAuthenticationChallenge: {
+            authChallenge = challenge;
+
+            if (Number(challenge.authenticationChallengeType) === 1) {
```

Comment:
Is this comparison using a number because the same view works against Cpp and QML and we can't use enum values in that case? Just making sure...